datacite-jsonapi-annotator/src/test/java/org/gbif/datacite/jsonapi/annotator/JsonApiAnnotatorTest.java

```diff
@@ -0,0 +1,64 @@
+package org.gbif.datacite.jsonapi.annotator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Type;
+import com.sun.codemodel.JAnnotationUse;
+import com.sun.codemodel.JDefinedClass;
+import com.sun.codemodel.JFieldVar;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class JsonApiAnnotatorTest {
```

Comment:
test cases names do not follow the camelCase format used for all the methods

Reply:
I usually use the test naming strategy testName_whenSomething_thenExpectSomethingElse. We used that on several past projects. 
If it's not ok I can change.
https://dzone.com/articles/7-popular-unit-test-naming

Reply:
in order to be consistent I will follow the naming strategy number 3 from the link above

Reply:
done